apps/src/componentLibrary/toggle/toggle.module.scss

```diff
@@ -0,0 +1,247 @@
+@import "color";
+
+.toggle {
+  display: flex;
+  flex-direction: row;
+  gap: 8px;
+
+  > span {
+    -webkit-user-select: none;
+    -moz-user-select: none;
+    -ms-user-select: none;
+  }
+
+  div {
+    display: inline-block;
+    position: relative;
+    vertical-align: middle;
+    text-align: left;
```

Comment:
I'm worried about how these left/right styles will look on pages with right-to-left languages. My assumption is that for a right-to-left language, the "off" would be on the right side and the "on" would be on the left side. Would the label left and label right also be switched? I set `direction=rtl` to see
<img width="555" alt="image" src="https://github.com/code-dot-org/code-dot-org/assets/9142121/d97956b2-1657-4283-8ea3-5d9e8afcfd91">
and it looks like there's no change in a right-to-left language

Reply:
Indeed, currently there're no styles for `rtl` languages. I need to discuss it with design team before implementing.
Also, do we have any `rtl` support for toggles currently in our codebase?

From what I see in new section screen - the only change for `rtl` language is the positioning of the label.

https://github.com/code-dot-org/code-dot-org/assets/22244040/61dc4eb5-4736-4f8a-94a1-5abba3ba733f

So it's possible that the `toggle right ` is supposed to be `rtl` languages styles. I'll discuss it with design team and change in this or new PR. Thanks for asking!

Reply:
if it's possible, maybe modeling this off of newer css values `start` and `end` rather than `left` and `right` could be helpful, for clear positioning in both LTR and RTL modes?

Reply:
@rshipp oh, that's great idea, thanks! I've spent some trying working with those attributes, but looks like it'll still require some `html[dir="rtl"] ` styles and manual adjustments because of a) `float: left`; b) `inset-block-start` and `inset-block-end` for `position absolute` are not replacing `left` and` right` _attributes_ correctly.

Anyways, I need to discuss it with designers first and then I'll make changes if they'll be approved. I will keep this comment in mind, but let's mark it as to be updated in other PR.

Reply:
I like that idea, Ryan. I don't feel comfortable shipping reusable components that aren't RTL-friendly –– but I also realize there's not a spec for this! I do think browsers should be able to take care of it if we use direction-agnostic styling (like `start`, `end`, `gap`, etc.)

Reply:
Again, it will be rtl-firendly. It's just not in the scope currently.